google/cloud/bigquery/table.py

```diff
@@ -1304,6 +1318,29 @@ def __init__(self, resource: Dict[str, Any]):
             )
 
 
+class CloneDefinition:
+    """Information about base table and clone time of the clone.
+
+    See https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#clonedefinition
```

Comment:
This is a nit and I love your attention to details, but do we need this link here? Or do we run the risk that it will go obsolete? I defer to @steffnay and/or @shollyman for what's typical with this library

Reply:
We have similar links to the docs in some other properties, including the `Table.snapshot` property, so I think it's okay to leave this as well.